open_widget_framework/open_widget_framework/widget_classes.py

```diff
@@ -47,7 +49,7 @@ def render(self):
         return format_html('<iframe src="{url}"></iframe>', url=self.data["url"])
 
 
-class ManyUserWidget(WidgetClassBase):
+class MembersWidget(WidgetClassBase):
```

Comment:
If you could just revert this back to its original name, it's more accurate as we'll have to implement membership widgets downstream in open.